operator-manifests/mattermost/operator.yaml

```diff
@@ -15,7 +15,7 @@ spec:
       serviceAccountName: mattermost-operator
       containers:
         - name: mattermost-operator
-          image: mattermost/mattermost-operator:v0.8.5
+          image: mattermost/mattermost-operator:0de3fa2
```

Comment:
On purpose?

Reply:
Oh no, not on purpose. Thanks for catching that.